client/src/index.jsx

```diff
@@ -1,12 +1,163 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import './styles.css';
+
+class App extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {number: 1};
+		this.addSighting = this.addSighting.bind(this);
+	}
+
+	renderSightings(){
+		const {number} = this.state;
+		if (!number | number < 0) return null;
+		let total = []
+		for (let i = 0; i < number; i+=1) {
+			total.push(
+				<SightingList user={person} sighting={sight} />
+			);
+		};
+		return total;
+	};
+
+	addSighting() {
+		console.log("Clicked Sighting");
+		this.setState(state => ({
+			number: state.number + 1
+		}));
+	}
+
+	render() {
+		return (
+			<div>
+				<h1>Ahmic Animals</h1>
+				<Header addSighting={this.addSighting} />
+				<h1>MAP GOES HERE</h1>
+				{this.renderSightings()}
+			</div>
+		);
+	};
+}		
+
+
+
+
+//Nav bar at top of page
+class Header extends React.Component {
+	constructor(props) {
+		super(props)
+		this.newSighting = this.newSighting.bind(this);
+	};
+
+	newSighting() {
+		this.props.addSighting();
+	};
+
+	render() {
+		return (
+			<span>
+				<button onClick={viewHome}>Home</button>
+				<button onClick={viewUser}>User</button>
+				<button onClick={viewAnimals}>Animals</button>
+				<button onClick={this.newSighting}>Add Sighting</button>
+			</span>
+		);
+	};
+}
+
+//Display user profile picture (if loaded)
+function Avatar(props) {
+	return (
+		<img className="Avatar"
+			src={props.user.avatarUrl}
+			alt={props.user.name}
+		/>		
+	);
+}
+
+//Include user's name below their profile picture
+function UserInfo(props) {
+	return (
+		<div className="UserInfo">
+			<Avatar user={props.user} />
+			<div className="UserInfo-Name">
+				{props.user.name}
+			</div>
+		</div>		
+	);
+}
+
+//SightingInfo
+function SightingInfo(props) {
+	return (
+		<div>
+			<h2>{props.sighting.animal}</h2>
+			<h4>{props.sighting.location}</h4>
+			<h4>{props.sighting.time}</h4>	
+		</div>	
+	);
+}
+
+//List version of a sighting
```

Comment:
This description doesn't make sense

Reply:
"list version" referred to list vs map displays, not the variable type.